docs/earthdata-varinfo.ipynb

```diff
@@ -198,6 +198,19 @@
    "source": [
     "In the output above, `/Grid/precipiationCal` has three dimensions: `/Grid/time`, `/Grid/lat` and `/Grid/lon`. These dimension variables also refer to their respective [bounds variables](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#cell-boundaries). Because `get_required_variables` is recursive, the bounds attributes are also considered required variables for `/Grid/precipitationCal`.\n",
     "\n",
+    "`earthdata-varinfo` checks for references in known CF Convention metadata attributes that are expected to contain references to other variables. These include:\n",
+    "\n",
+    "* [ancillary_variables](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#ancillary-data)\n",
+    "* [coordinates](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#coordinate-types), see also [here](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#coordinate-system).\n",
+    "* [grid_mapping](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#grid-mappings-and-projections)\n",
+    "* [bounds](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#cell-boundaries)\n",
+    "* [cell_measures](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#cell-measures)\n",
+    "* [geometry, interior_ring, node_coordinates, node_count, nodes and part_node_count](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#geometries)\n",
+    "\n",
+    "The check is recursive, as a variable referred to in the initial set specified in the call to the `get_required_variables` method might itself refer to other variables.\n",
```

Comment:
You do mention it's recursive right above this change also, I don't think it's a big deal, but wasn't sure if you wanted to reword a bit?  Maybe `Because the check is recursive,` would be enough

Reply:
I'll make that change and push it up with the potential other change being discussed, once we settle on wording there.

Reply:
Looking at it, and the content from before, is this sentence actually adding anything? Could we just cut it in favour of the comment above about how the `bounds` variables are included.

Reply:
I don't know that was my first thought but then again people skim documentation so repeating isn't terrible.

Reply:
I couldn't think of a good way to make two entirely distinct points here, so I've cut it in [b50c3d](https://github.com/nasa/earthdata-varinfo/pull/35/commits/b50c3da3c9d2e4ecb51a76f4003b0100e9cd34fd).